CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ cd dnsimple-java
 
 The project includes an [ASDF](https://github.com/asdf-vm/asdf) `.tool-versions` file to set up the JVM required to work on the project.
 
-You can install the required version of Java executing `asdf install` at the root directory of the project.
+You can install the required version of Java executing `asdf install` at the project's root directory.
```

Comment:
It may require an English confirmation, but project is not a person so I believe the former version was more correct.

Reply:
Grammarly doesn't complain about that one. Maybe @itsalyse can weigh in.

Reply:
The second one is good 👍 - objects can take the possessive form, and it usually makes for tighter copy.